client/pages/reading-list.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+export default class ReadingList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: null
+    };
+  }
+
+  componentDidMount() {
+    fetch('/api/bookShelf/readingList')
+      .then(res => res.json())
+      .then(result => {
+        this.setState({ result });
+      })
+      .catch(error => console.error(error));
+  }
+
+  render() {
+    const { result } = this.state;
+    if (!result) return null;
+    const books = result;
+    const bookResults = (
+      <div className="reading-list-container">
+        {
+          books.map((book, index) => {
+            const title = book.title;
+            const thumbNail = book.coverUrl;
+            const googleId = book.googleId;
+            const author = book.author;
+            return (
+              <div key={index} id={googleId} className="reading-list-card">
```

Comment:
If your google ids are unique, you should use those as the key instead of index.